README.md

```diff
@@ -18,6 +81,43 @@ Non exaustive list:
 - pkg-config
 - C Compiller
 
-## contribution
+
+## Contribution
 
 Open Pull Requests and Issues are welcome.
+
+If you want to add binary in bin/ please try to make it easy to compile.
```

Comment:
If you want to add binaries to the `bin/` directory, please ensure they are easy to compile.